src/lib/components/tablev2/MultiSelectableContent.tsx

```diff
@@ -41,6 +41,8 @@ type MultiSelectableContentProps<
   DATA_ROW extends Record<string, unknown> = Record<string, unknown>,
 > = {
   onMultiSelectionChanged: (rows: Row<DATA_ROW>[]) => void;
+  onSingleRowSelected: (row: Row<DATA_ROW>) => void;
+  onToggleAll: (selected: boolean) => void;
```

Comment:
I guess those callbacks should be optional
```suggestion
  onSingleRowSelected?: (row: Row<DATA_ROW>) => void;
  onToggleAll?: (selected: boolean) => void;
```